### ISSUE TITLE
ARIA 1.1: aria-level is required with role="heading" 

### ISSUE THREAD

**Issue by jsphstls** [gpbl/react-day-picker#922]
Hello and thank you for this excellent package!

I ran `axe-core` testing against a component of mine that uses this package and encountered this error:
```
    Expected the HTML found at $('.DayPicker-Caption') to have no violations:

    <div class="DayPicker-Caption" role="heading"><div>September 2019</div></div>

    Received:

    "Required ARIA attributes must be provided (aria-required-attr)"

    Fix any of the following:
      Required ARIA attribute not present: aria-level

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/3.3/aria-required-attr?application=axeAPI
```

I realize that this could a be a fault of `axe-core` instead of `react-day-picker`. The key issue is that while an element with `role="heading"` should have an implicit `aria-level` of 2, the outcome is not consistent and the attribute is marked as required. Even if `aria-level=2` was intentional, it may not be correct depending on the context where the heading ends up.

Some useful discussion around the implementation of this rule is available here:
https://github.com/dequelabs/axe-core/issues/814

I am not sure what the ideal fix would be. Maybe the `heading` role is replaced with another technique or `aria-level` is included?

**Comment by reintroducing** [comment]
I came here to report the same exact issue so I'm glad I looked through the issues before doing so. I'm using the `addon-a11y` in Storybook while developing a wrapper around this component and am seeing the same thing (understandably so since it uses axe under the hood). 

@gpbl I think you said you'll be using Storybook for the next release so I'd highly suggest adding that addon to help in the a11y of this awesome module. In the meantime, what can be done about this to alleviate this error?

**Comment by jsphstls** [comment]
After some more reading on the `heading` role I see that it's really a substitute for `h1-6`. I don't think a heading element or role is needed here at all. The easiest fix is to remove it.

After that I wonder what should be used instead. Maybe an `aria-labelledby` relationship?

**Comment by reintroducing** [comment]
I think another possible solution is similar to whats outlined here: https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/datepicker-dialog.html

Can use an aria-live attribute instead.

**Comment by ArcadeRenegade** [comment]
+1
I am getting an error in Lighthouse about this.

**Comment by jsphstls** [comment]
I was looking into a fix for this but I an unable to find a place to make a change:
https://github.com/gpbl/react-day-picker/search?q=heading&unscoped_q=heading

Perhaps this was already fixed?

**Comment by gpbl** [comment]
@jsphstls sadly the search doesn't work in branches. The attribute you are looking for is here: https://github.com/gpbl/react-day-picker/blob/v7/src/Caption.js

**Comment by AlberErre** [comment]
Hi, I'm facing the same issue as  @jsphstls  

In my case the CI doesn't allow me to deploy because my `jest-axe` test fails by finding this ARIA related error.

🚨 The issue is over [here](https://github.com/gpbl/react-day-picker/blob/v7/src/Caption.js), at line 54, we should add either remove the `role="heading"` , or add an extra prop `aria-level="1"`.

@gpbl, I would go with the second option by adding `aria-level="1"`, as it would resolve the issue without losing accessibility.

**Comment by gpbl** [comment]
Closing as [we've released v8](https://github.com/gpbl/react-day-picker/discussions/1412): this issue belongs to a version which is not longer maintained, and it is likely fixed. In case, please open a new issue. Thanks!
